tap/tlstapper/tls_poller.go

```diff
@@ -195,36 +212,76 @@ func (p *tlsPoller) closeReader(key string, r *tlsReader) {
 	p.closedReaders <- key
 }
 
-func buildTlsKey(chunk *tlsChunk, ip net.IP, port uint16) string {
-	return fmt.Sprintf("%v:%v-%v:%v", chunk.isClient(), chunk.isRead(), ip, port)
+func (p *tlsPoller) getAddressPair(chunk *tlsChunk) (addressPair, error) {
```

Comment:
This signature is a little misleading. It looks like this function is getting the chunk address, but actually it is getting the process's address.
`getAddressPair(pid, fd)` or `getProcessAddressPair(chunk)` is better I think.